web/src/LuksActivationQuestion.jsx

```diff
@@ -39,13 +39,21 @@ const renderAlert = (attempt) => {
 export default function LuksActivationQuestion({ question, answerCallback }) {
   const [password, setPassword] = useState(question.password || "");
   const conditions = { disable: { decrypt: password === "" } };
+  const defaultAction = "decrypt";
 
   const actionCallback = (option) => {
     question.password = password;
     question.answer = option;
     answerCallback(question);
   };
 
+  const triggerDefaultAction = async (e) => {
```

Comment:
NP: since y the function does not contain an `await` statement, I think `async` is not needed here. Although technically it _isn't wrong_ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#description.

Reply:
oops, copy&paste :-)